sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -172,6 +173,12 @@ export function createEnvironmentCredential(
   return new EnvironmentCredential(options);
 }
 
+export function createDefaultVsCodeCredential(
+  options: DefaultAzureCredentialOptions = {},
+): TokenCredential {
+  return new VisualStudioCodeCredential(options);
```

Comment:
Consider setting brokerEnabled : true here

or... you will have to set it in the VsCodeCredential instead

Reply:
Yes, DAC options should be the overload of all the credential option and `VisualStudioCodeCredentialOptions` will have the `brokerOptions`